src/main/java/io/github/oliviercailloux/y2018/jconfs/Conference.java

```diff
@@ -108,11 +111,12 @@ public String getCountry() {
 	/**
 	 * This is a setter which set the country
```

Comment:
Tant que vous y êtes, retirez ce genre de commentaire inutile. Ou indiquez de façon plus courte : `Sets the country.`